overtime_analyzer.py

```
@@ -0,0 +1,941 @@
+                    df[col_mapping["발생일자"]], errors="coerce"
+                )
+
+            # 시각 열이 문자열이면 datetime.time으로 변환
+            if not pd.api.types.is_datetime64_any_dtype(df[col_mapping["발생시각"]]):
+                try:
+                    df["시간_datetime"] = pd.to_datetime(
+                        df[col_mapping["발생시각"]], errors="coerce"
+                    )
+                    df["시간_시"] = df["시간_datetime"].dt.hour
+                    df["시간_분"] = df["시간_datetime"].dt.minute
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)